Src/BootCamp.Chapter/TransactionStream.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    internal class TransactionStream : IDisposable
+    {
+        private StreamReader _reader;
+        private Transaction _current;
+
+        public TransactionStream(Stream stream)
+        {
+            _reader = new StreamReader(stream);
+            IgnoreHeaderRow();
+        }
+
+        private void IgnoreHeaderRow() => _ = _reader.ReadLine();
```

Comment:
I think this method is overengineered, because it ever only works when the reader hasn't read any lines yet. In case it had, it will have the opposite behaviour. Maybe it should be renamed simply to: ``IgnoreRow()``?